### ISSUE TITLE
Verifier is unable to gracefully shutdown when running as frontend

### ISSUE THREAD

**Issue by kaifeng** [keylime/keylime#492]
When running keylime_verifier as a frontend process, there is a high change that verifier can't be killed by Ctrl+C, and next execution of verifier will complain address is in use.

```
[root@Controller-1 ~]# ps aux | grep keylime_verifier
root      998610  0.8  0.0 249744 51916 pts/8    S    16:52   0:00 /usr/bin/python3 /usr/bin/keylime_verifier
root      999685  0.0  0.0  12108  1080 pts/8    S+   16:52   0:00 grep --color=auto keylime_verifier
```

Tried to call server.stop() but doesn't address the issue, needs more digging.

**Comment by kaifeng** [comment]
Possible cause: https://github.com/tornadoweb/tornado/issues/1912

**Comment by ansasaki** [comment]
@THS-on Hello, should this be closed via #889?

@kaifeng Would it be possible to confirm if this is fixed for you?

**Comment by THS-on** [comment]
@ansasaki yes, should be also fixed with #889.
